nautobot/dcim/views.py

```diff
@@ -1132,41 +1132,31 @@ def import_view(self, request, *args, **kwargs):
 #
 # Console port templates
 #
-
-
 class ConsolePortTemplateCreateView(generic.ComponentCreateView):
     queryset = ConsolePortTemplate.objects.all()
     form = forms.ConsolePortTemplateCreateForm
     model_form = forms.ConsolePortTemplateForm
     template_name = "dcim/device_component_add.html"
```

Comment:
What's the plan for integrating this into the UIViewSet? Probably need a new viewset mixin class equivalent to `ComponentCreateView`.

Reply:
It wasn't clear to me, so documenting here, the request is to replace [this view](https://github.com/nautobot/nautobot/blob/38a91b59acb237e3806255fe914309aabd7be552/nautobot/core/views/generic.py#L1307) as a mixin.